src/api/lib/tasks/dev/requests.rake

```diff
@@ -96,5 +96,45 @@ namespace :dev do
 
       puts "* Request #{request_action.bs_request.number} has been created."
     end
+
+    desc 'Creates a request which builds and produces build results'
+    task request_with_build_results: :environment do
+      unless Rails.env.development?
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        puts 'otherwise it will destroy your database data.'
+        return
+      end
+
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      puts 'Creating a request which builds and produces build results...'
+      admin = User.get_default_admin
+      home_admin_project = RakeSupport.find_or_create_project(admin.home_project_name, admin)
+
+      # Branch the hello_world package
+      iggy = User.find_by(login: 'Iggy') || create(:staff_user, login: 'Iggy')
+      branches_iggy = Project.find_by(name: iggy.branch_project_name(home_admin_project.name)) || create(:project, name: iggy.branch_project_name(home_admin_project.name))
+      hello_world_iggy = create(:package, name: "hello_world_#{Faker::Lorem.word}", project: branches_iggy)
+      backend_url = "/source/#{CGI.escape(branches_iggy.name)}/#{CGI.escape(hello_world_iggy.name)}"
+      hello_world_spec = File.read('../../dist/t/spec/fixtures/hello_world.spec')
+      hello_world_spec.gsub('Most simple RPM package', Faker::Lorem.sentence(word_count: 4))
+      Backend::Connection.put("#{backend_url}/hello_world.spec", hello_world_spec)
+
+      # Create the request
+      request = create(
+        :bs_request_with_submit_action,
+        creator: iggy,
+        target_project: home_admin_project,
+        target_package: 'hello_world',
+        source_project: branches_iggy,
+        source_package: hello_world_iggy
+      )
+      puts "* Request #{request.number} has been created."
+      puts 'To start the builds confirm or perfom the following steps:'
+      puts '- Create the interconnect with openSUSE.org'
+      puts '- Create a couple of repositories in project home:Iggy or in project home:Iggy:branches:home:Admin'
```

Comment:
```suggestion
      puts '- Create a couple of repositories in project home:Iggy:branches:home:Admin'
```

As we do the branch before we add the repos to home:Iggy, those repos are not copied to the branched project. So it doesn't build anything.

Reply:
Ops, true. Changed.